### PR TITLE
Described repository index layout

### PR DESCRIPTION
**Pull request by olenagerasimova**
Added badges to readme, formatted it as all the readmes in other adapters and described repository index layout.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,65 @@
...
+where `dl` is the url for downloading crates and `api` is the base URL for the web API, 
+`api` is optional, but without it commands such `cargo publish` will not work.
 
-Sources of crates.io: https://github.com/rust-lang/crates.io
+Download url should send `.crate` file for the requested package, package name and version are appended 
+to the base url along with `downlowd` word: `{package_name}/{package_version}/download`. `dl` value
+also may have markers in the url, find more information [here](https://doc.rust-lang.org/cargo/reference/registries.html#index-format).
 
-Start Crates with docker-compose: https://github.com/rust-lang/crates.io/blob/master/docs/CONTRIBUTING.md#running-cratesio-with-docker
+The rest of the repository index contains one file for each package, where each version metadata are
```
Comment (genryxy) [inline README.md:44]: typo for `each version metadata are present` --> `each version metadata is presented`

File: README.md [hunk 2]
```diff
@@ -1,16 +1,65 @@
...
+    "api": "https://crates.io"
+}
+```
+where `dl` is the url for downloading crates and `api` is the base URL for the web API, 
+`api` is optional, but without it commands such `cargo publish` will not work.
 
-Sources of crates.io: https://github.com/rust-lang/crates.io
+Download url should send `.crate` file for the requested package, package name and version are appended 
+to the base url along with `downlowd` word: `{package_name}/{package_version}/download`. `dl` value
+also may have markers in the url, find more information [here](https://doc.rust-lang.org/cargo/reference/registries.html#index-format).
```
Comment (genryxy) [inline README.md:42]: typo in the word `downlowd`

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, two minor comments

**Comment by olenagerasimova** [comment]
@genryxy thanks

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**